compiler/luci-interpreter/src/loader/GraphLoader.cpp

```diff
@@ -88,7 +107,19 @@ bool isExecutableNode(const luci::CircleNode *node)
     case luci::CircleOpcode::CIRCLESPLITVOUT:
     case luci::CircleOpcode::CIRCLEUNPACKOUT:
     case luci::CircleOpcode::CIRCLEWHILEOUT:
+    case luci::CircleOpcode::CIRCLECUSTOMOUT:
       return false;
+    // Custom nodes may be executable and non-executable
+    case luci::CircleOpcode::CUSTOM:
+    {
+      auto const custom_node = loco::must_cast<const luci::CircleCustom *>(node);
+
+      // TODO handle more custom ops here
+      if (custom_node->custom_code() == "CircleReferencingConst")
+        return false;
+
+      throw std::runtime_error("Unknown Custom Node, yet.");
```

Comment:
If we throw here, maybe also check and throw inside `isTensorProducingNode()` for consistancy?

Reply:
Hmm, this suggestion makes sense, but I don't like idea of adding more checks for supported `CircleCustom` cases. So, I think it would be better to add another helper: `isValidCustomNode` and check Custom node is valid only once in thi s helper.

Reply:
And therefore remove this throw here.

Reply:
> So, I think it would be better to add another helper: `isValidCustomNode` and check Custom node is valid only once in thi s helper.

I added this check with name "isSupportedCustomNode".